src/scss/form/button.scss

```diff
@@ -40,6 +40,8 @@
 .button.blank:not(.disabled):hover,
 .button.blank:not(.disabled).hover { background: var(--color-shape-highlight-medium); }
 
+.button.active { background: var(--color-shape-highlight-medium); }
```

Comment:
Incorrect placement, should be only inside .pageMainDate it will affect all buttons globally.